src/main/java/com/github/splendor_mobile_game/websocket/handlers/reactions/StartGame.java

```diff
@@ -101,9 +262,290 @@ public StartGame(int connectionHashCode, UserMessage userMessage, Messenger mess
+            this.rubyMinesRequired = rubyMinesRequired;
+            this.emeraldMinesRequired = emeraldMinesRequired;
+            this.sapphireMinesRequired = sapphireMinesRequired;
+            this.diamondMinesRequired = diamondMinesRequired;
+            this.onyxMinesRequired = onyxMinesRequired;
+        }
+    }
+
+    public class MinesCardDataResponse{
+        public UUID uuid;
+        public int prestige;
+        public Color color;
+
+        public int rubyTokensRequired;
+        public int emeraldTokensRequired;
+        public int sapphireTokensRequired;
+        public int diamondTokensRequired;
+        public int onyxTokensRequired;
+
+        public MinesCardDataResponse(UUID uuid, int prestige, Color color, int rubyTokensRequired,
+                int emeraldTokensRequired, int sapphireTokensRequired, int diamondTokensRequired, int onyxTokensRequired) {
+            this.uuid = uuid;
+            this.prestige = prestige;
+            this.color = color;
+            this.rubyTokensRequired = rubyTokensRequired;
+            this.emeraldTokensRequired = emeraldTokensRequired;
+            this.sapphireTokensRequired = sapphireTokensRequired;
+            this.diamondTokensRequired = diamondTokensRequired;
+            this.onyxTokensRequired = onyxTokensRequired;
+        }
+        
+    }
+
+    public class UserDataResponse {
+        public UUID uuid;
+
+        public UserDataResponse(UUID uuid) {
+            this.uuid = uuid;
+        }      
+    }
+
+    public class ResponseData{
+        public TokensDataResponse tokens;
+        public ArrayList<NobleDataResponse> nobles;
+
+        public ArrayList<MinesCardDataResponse> firstLevelMinesCards;
+        public ArrayList<MinesCardDataResponse> secondLevelMinesCards;
+        public ArrayList<MinesCardDataResponse> thirdLevelMinesCards;
+
+        public UserDataResponse userToPlay;
+
+        public ResponseData(TokensDataResponse tokens,
+                ArrayList<NobleDataResponse> nobles, ArrayList<MinesCardDataResponse> firstLevelMinesCards,
+                ArrayList<MinesCardDataResponse> secondLevelMinesCards,
+                ArrayList<MinesCardDataResponse> thirdLevelMinesCards,
+                UserDataResponse userToPlay) {
+            this.tokens = tokens;
+            this.nobles = nobles;
+            this.firstLevelMinesCards = firstLevelMinesCards;
+            this.secondLevelMinesCards = secondLevelMinesCards;
+            this.thirdLevelMinesCards = thirdLevelMinesCards;
+            this.userToPlay=userToPlay;
+        }
+        
+    }
+
+
+
+
+
     @Override
     public void react() {
-        // TODO Auto-generated method stub
+        DataDTO dataDTO = (DataDTO)userMessage.getData();
+
+        try{
+            //validateData(dataDTO, database);
+
+            User user = new User(dataDTO.userDTO.uuid, "janek", connectionHashCode);
+            Room room = new Room(dataDTO.roomDTO.uuid, "pokoj", "123", user, database);
```

Comment:
Oh my goodness, I didn't notice that! This should be retrieved from the database, not created. xDDD